localization/english/lifestyles/ancient_magic_perks_l_english.yml

```diff
@@ -9,7 +9,7 @@
  mage_perk_name: "Mage"
 
  # chronomancy magic
- time_mage_perk_name: "Chronomancer"
+ chronomancy_mage_perk_name: "Chronomancer"
```

Comment:
nice, but weird because i though i renamed it already

Reply:
Yeah, this is another one of those instances where one of us did something, and a commit from the other one changed it back for some reason. Like with `none` and `basic` a few days ago.